qutebrowser/browser/commands.py

```diff
@@ -672,8 +673,11 @@ def yank(self, title=False, sel=False, domain=False):
                                    ':' + str(port) if port > -1 else '')
             what = 'domain'
         else:
-            s = self._current_url().toString(
-                QUrl.FullyEncoded | QUrl.RemovePassword)
+            cur_url = self._current_url()
+            if pretty:
+                s = cur_url.toString(QUrl.RemovePassword)
+            else:
+                s = cur_url.toString(QUrl.FullyEncoded | QUrl.RemovePassword)
```

Comment:
I think this would be (marginally) cleaner:

```python
flags = QUrl.RemovePassword
if not pretty:
    flags |= QUrl.FullyEncoded
s = cur_url.toString(flags)
```